core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java

```diff
@@ -246,7 +246,7 @@ public TarEntry getNextEntry() throws IOException {
             this.currEntry = new TarEntry(headerBuf);
 
             if (this.debug) {
-                System.err.println("TarInputStream: SET CURRENTRY '"
+                System.err.println("TarInputStream: SET currENTRY '"
```

Comment:
To match: https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java#L215
```java
                System.err.println("TarInputStream: SKIP currENTRY '"
```